sw_i2c/sensirion_sw_i2c.c

```diff
@@ -37,12 +37,13 @@
 #define DELAY_USEC (SENSIRION_I2C_CLOCK_PERIOD_USEC / 2)
 
 static int8_t sensirion_wait_while_clock_stretching(void) {
-    uint8_t timeout = 100;
+    /* Maximal timeout of 150ms (SCD30) in sleep polling cycles */
+    uint32_t timeout = 150000 / SENSIRION_I2C_CLOCK_PERIOD_USEC;
```

Comment:
Can we rename to `timeout_cycles` since timeout has the notion of time..

Reply:
Done